### DISCUSSION TITLE
It would help to have some kind of "relationship parents" field that would display entries that are parents

### DISCUSSION THREAD

**Discussion by MikeTheVike** [ExpressionEngine/ExpressionEngine#2993] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
If I have a news channel and a channel for pages and I want to put a news article on a page, I could put the relationship field on the news channel or the pages channel. It would make sense, and be one less step for content editors, if they could select the page in a relationship field on the news entry form. BUT, I frequently have clients that want to change the order the articles display on the pages with manual reordering. So that means I need to put the relationship field on the pages channel. This works for what they want, but requires them to add a new news entry, then open up the pages entry to add the news entry to the page. Maybe there could be some kind of “relationship parent” field that displays entries that have been assigned to it. Then you could put the relationship field on the news channel, put the “relationship parent” field on the pages channel, and it would display news entries assigned to it. Then you could also reorder if desired.

**Describe the solution you'd like**
I'm thinking it might be a field that lists the entries that have chosen that entry in a relationship field. You would be able to drag and drop to reorder. Not sure how it would work for entries with multiple parents, maybe it divide the lists by channel?

**Describe alternatives you've considered**

**Teachability, Documentation, Adoption, Migration Strategy**

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by robinsowell** [comment]
While I do like the idea of being able to at least see the entries an entry is related to from the edit, it kind of feels like the relationship field belongs on the pages entry.

They should be able to add a new news article directly from pages- there's an 'add entry' button at the bottom of a relationship field that lets you create an entry on the fly and have it added as a relationship selection on the page you're on.

<img width="660" alt="Screen Shot 2019-08-21 at 4 24 32 PM" src="https://user-images.githubusercontent.com/1181219/63466005-54b2f780-c430-11e9-82cd-ef93307a0a5c.png">

Or am I misunderstanding what you're going for?

(And I do still like the display idea.)

**Comment by MikeTheVike** [comment]
Robin, you are probably right that training our client to select a page first and then adding a new entry from there might be the best bet. I rarely use that feature myself when developing, so it didn't occur to me. Thanks!
Mike
    On Wednesday, August 21, 2019, 03:26:21 PM CDT, robinsowell <notifications@github.com> wrote:  
 
 
While I do like the idea of being able to at least see the entries an entry is related to from the edit, it kind of feels like the relationship field belongs on the pages entry.

They should be able to add a new news article directly from pages- there's an 'add entry' button at the bottom of a relationship field that lets you create an entry on the fly and have it added as a relationship selection on the page you're on.



Or am I misunderstanding what you're going for?

(And I do still like the display idea.)

**Comment by robinsowell** [comment]
I REALLY wanted that feature- I used to run into it all the time.  

I do still agree at least being able to see existing parents would be nice, but I think UI would need to be worked out for that.

**Comment by robinsowell** [comment]
Noting that for now, Hop Introvert is definitely worth a look! https://expressionengine.com/add-ons/hop-introvert

**Comment by intoeetive** [comment]
I think "reverse relationships" read-only field totally makes sense. It can have the UI similar to Relationship when in "defer" mode (so probably no React component is necessary) but without button to edit the relationship.

It should also have links to edit the reverse related entries in a slide-out, like we do with related entries

The field itself will hold no data, and will be populated dynamically. It will only be available in the CP, and similarly to Notes fieldtypes provide no front-end tags

The necessary setting for field is "Relationship fields" - give ability to display entries related by a specific field, or few fields, or any field. We might also want to keep "display entry_id / channel / status" settings

**Reply by intoeetive** [reply]
Another option is to have a tab - similar to "Usage" tab that we have in File Manager - that would list "what links here", i.e. entries that are relating to this one
